content/projects/remoting/index.adoc

```diff
@@ -23,7 +23,7 @@ The Remoting sub-project includes the Remoting library itself, package for agent
 == Remoting Usage in Jenkins
 
 * Agent executables (`remoting.jar` is named `agent.jar` or `slave.jar` in Jenkins)
-* Controller to Agent communication protocols being used in various Agent types, including inbound (JNLP) and plugin:ssh-slaves[SSH] agents
+* Controller to Agent communication protocols being used in various Agent types, including inbound (formerly known as “JNLP”) and plugin:ssh-slaves[SSH] agents
```

Comment:
Scare quotes are appropriate here (and used elsewhere in the documentation when referring to “JNLP”), since JNLP isn't used at all anymore except in deprecated code paths.